crates/hash/src/multi_digest.rs

```diff
@@ -172,16 +172,71 @@ impl<D: Digest + Send + Sync + Clone> ParallelDigest for D {
 	}
 }
 
+#[derive(Clone)]
+pub struct MultipleDigests<D: Digest, const N: usize>([D; N]);
```

Comment:
Do we need this? `ParallelDigest` is sufficient for merkle tree integration. And for the purposes of benchmarking, we can compare regular digest throughput to Multidigest throughput.